competitor.php

```diff
@@ -11,172 +11,58 @@
 <a href="http://www.cubingusa.com/ctutorial.php">Competitor Tutorial</a>
 von <a href="http://www.cubingusa.com">Cubing USA</a>. <br/>
 Übersetzt und ergänzt von <a href="http://hackvalue.de">Florian Weingarten</a>
-(<a href="http://www.worldcubeassociation.org/results/p.php?i=2007WEIN01">2007WEIN01</a>)<br/>
-<a href="http://aachen.speedcubing.com/">
-http://aachen.speedcubing.com/</a>
+(<a href="http://www.worldcubeassociation.org/results/p.php?i=2007WEIN01">2007WEIN01</a>) und vielen anderen.
 </p>
 
-<? include "credits.php" ?>
-
 <? include "menu.php" ?>
 
 <p>
-Du möchtest an deinem ersten Speedcubing-Wettbewerb ("Competition")
-teilnehmen, aber weißt nicht wie es dort so abläuft? Diese Seite
-soll dir eine kleine Hilfe sein, damit deine erste Competition
-ein schönes Erlebnis wird. Jede Competition ist anders, aber ein
-paar Grundabläufe sind bei fast allen Competitions gleich.
+Du möchtest an deinem ersten Speedcubing-Wettbewerb ("Competition") teilnehmen, aber weißt nicht wie es dort so abläuft? Diese Seite soll dir eine kleine Hilfe sein, damit deine erste Competition ein schönes Erlebnis wird. Jede Competition ist anders, aber die Grundabläufe sind bei allen Competitions gleich, da sie stets nach dem WCA Regelwerk ablaufen. Die WCA ist eine internationale Organisation für das Speedcubing, etwa wie die FIFA für den Fußball. Auf jedem Turnier ist mindestens ein offizieller WCA Delegierter anwesend, der die Einhaltung der Regeln überwacht und stets gerufen werden sollte, wenn jemand nicht genau weiß, was zu tun ist.
```

Comment:
those super long lines make it really annoying to proofread the text here on GitHub... Any chance you could insert some line breaks (at around 80-100 characters width)? That would help a lot.